src/locales/en/translation.json

```diff
@@ -1649,6 +1649,7 @@
     },
     "liquidClaimForm": {
       "note": "Claiming your rewards automatically adds them to your SOV portfolio balance.",
+      "txNote": "You may need to claim multiple times to receive your rewards.",
```

Comment:
I think the message should be a bit different - https://sovryn.atlassian.net/browse/SOV-2307?focusedCommentId=16751

Reply:
Yeah, it's weird how it's back, fixed